src/Akeneo/Platform/Bundle/InstallerBundle/Event/Subscriber/RefreshVersioningSubscriber.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Akeneo\Platform\Bundle\InstallerBundle\Event\Subscriber;
```

Comment:
```suggestion
namespace Akeneo\Platform\Bundle\InstallerBundle\Event\Subscriber;
```

In a decoupled application, it should be part of the versioning code, not installer.